Assignment5/Assignment5.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+
+const makePromise = (fileName, fileOperation, tempData) => {
+  return new Promise((resolve, reject) => {
+    fs[fileOperation](fileName, tempData, (error, dataFromFile) => {
+      error ? reject(error) : resolve(dataFromFile);
+    })
+  })
+}
+
+const print = (string) => {
+  console.log(string);
+}
+
+makePromise('file1.txt', "readFile", "")
```

Comment:
Remove the "" (empty string), if you will not pass empty strings, it will be treated as `undefined`.